pkg/ext/stores/tokens/tokens.go

```diff
@@ -225,6 +226,8 @@ func (t *Store) Create(
 	obj runtime.Object,
 	createValidation rest.ValidateObjectFunc,
 	options *metav1.CreateOptions) (runtime.Object, error) {
+	logrus.Infof("ext.cattle.io/token create")
```

Comment:
Should these logs be `debug` otherwise there's a lot of them?

Reply:
Debug is likely better. Working on it now.